openreview/venue/venue.py

```diff
@@ -418,6 +421,9 @@ def update_submission_readers(submission):
                     )
         ## Release the submissions to specified readers if venueid is still submission
         openreview.tools.concurrent_requests(update_submission_readers, submissions, desc='update_submission_readers')
+
```

Comment:
can we add adding this venue to `active_venues` somewhere in the `post_submission_stage()`?

Reply:
good point, I will add it

Reply:
Done.